src/detail/utils.cpp

```diff
@@ -24,7 +24,8 @@
 
 
 uint32_t
-fastrtps__size_t_to_uint32_t(size_t in)
+fastdds__size_t_to_uint32_t(
+  size_t in)
```

Comment:
This seems like an unnecessary whitespace change:

```suggestion
fastdds__size_t_to_uint32_t(size_t in)
```

(the same goes for most of the functions in this file)